yarn.lock

```diff
@@ -2,946 +2,1065 @@
 # yarn lockfile v1
 
 
-"@babel/code-frame@7.5.5":
```

Comment:
I assume all of these changes comes from deduping?

Reply:
Yes, I think so. Some changes are also due to the analytics upgrade